### PR TITLE
Elgg tabs js

### PR DESCRIPTION
**Pull request by jeabakker**

### CODE REVIEW

File: views/default/page/components/tabs.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ define(function (require) {
 				data: $link.data('ajaxQuery') || {},
 				beforeSend: function () {
 					changeTab($tab);
+					$target.html('');
```
Comment (hypeJunction) [inline views/default/page/components/tabs.js]: Are you sure about this?

Comment (jeabakker) [reply views/default/page/components/tabs.js]: yes, after the ajax call is done, eighter it gets an error text of the result of the ajax call. SO no matter what the html content gets replaced by something else

Comment (hypeJunction) [reply views/default/page/components/tabs.js]: Maybe disable spinner and populate with ajax loader instead?

Comment (jdalsem) [reply views/default/page/components/tabs.js]: i believe this is correct behaviour for page/components/tabs

**Pull request merged**